README.md

```diff
@@ -33,10 +33,10 @@ pip install molecule-gce
 
 If you plan on testing Windows instances you also need pywinrm:
 ```
-pip install pywinrm
+pip install pywinrm google-api-python-client oauth2client pycryptodome
```

Comment:
Just remove instruction about python dependencies, these should be part of package requirements (implicit).

Keep only `pip3 install "molecule-gce[windows]"` as we should have a windows extra that lists the windows specific requirements.

On the other hand I am wondering if we should not just include everything with normal driver, also windows requirements. Do you see a problem with that?

Reply:
Ok, will delete all instructions about dependencies.

No problem with that

Reply:
I applied the required changes